lua/entities/gmod_wire_expression2/core/custom/wiring.lua

```diff
@@ -114,3 +114,33 @@ e2function wirelink entity:wirelink()
 	end
 	return this
 end
+
+__e2setcost(10)
+
+--- Links <this> to <ent2> if applicable
+e2function number entity:linkTo(entity ent2)
+	if not IsValid(this) then return self:throw("Invalid Entity!", nil) end
```

Comment:
all of the throws need to have 0 as the second parameter, not nil, because the function must return a number